docs/source/other/index.rst

```diff
@@ -7,3 +7,4 @@ Other Documentation
 
     faq
     setup_virtual_machine
+    uninstallation
```

Comment:
Not needed now that uninstallation is covered in the installation guide, but perhaps make a link to it in the FAQ just to make it more apparent where it is

Reply:
Thoughts on linking to it in the FAQ @courtneycb ?